packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -572,4 +572,9 @@ export enum CacheTagType {
     subscriptions = 'subscriptions',
 }
 
+export type LoadingState = {
+    isLoading?: boolean;
+    isFetching?: boolean;
+};
```

Comment:
I think an enum would be better since you can specifically name the states:
`Loading`, `RefreshingData`, `Done`. Also the 2 booleans can represent 4 states, but i can only name 3. This means that one combination of the booleans isLoading and isFetching is invalid.

Reply:
The main argument I have against this is that when checking the value now you can do ```
```if(loadingState.isLoading)``` and with an enum you would have to do '''loadingState === LoadingState.Loading``` In any case, I think this is really detailed and opinionated stuff and a bit much to hold up this issue. I prefer to merge it now.